var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -197,20 +203,30 @@ def install(self, spec, prefix):
                 juliarc.write('push!(Libdl.DL_LOAD_PATH, "%s")\n' %
                               spec["hdf5"].prefix.lib)
                 juliarc.write('\n')
-            julia("-e", 'Pkg.add("HDF5"); using HDF5')
-            julia("-e", 'Pkg.add("JLD"); using JLD')
+            if self.spec.version >= Version('1.0.0'):
```

Comment:
@baberlevi it appears that all `julia` invocations after this point differ only in terms of the `using Pkg;` prefix. The logic could potentially be consolidated by prepending this for `julia` versions >= `1.0.0` (potentially with a function that constructs the `Pkg.add` command from the package name and `julia` version).

I'm merging this anyhow but thought I'd add a note for future edits to `julia`.

Reply:
Thanks for the feedback, that's a good idea. I'll work on that and ping you to re-review